web-marketplace/src/components/organisms/Stakeholders/Stakeholders.tsx

```diff
@@ -18,62 +22,92 @@ export type Stakeholder = {
 
 type Props = {
   stakeholders: Stakeholder[];
+  className?: string;
 };
 
-export const Stakeholders = ({ stakeholders }: Props) => {
+export const Stakeholders = ({ stakeholders, className }: Props) => {
   const { _ } = useLingui();
 
   const filtered = stakeholders.filter(u =>
     Array.isArray(u.accounts) ? u.accounts.length > 0 : Boolean(u.accounts),
   );
   const sm = filtered.length === 4 ? 3 : 4;
 
+  const hasStakeholders = stakeholders.some(
```

Comment:
We could memoize this as it runs on every render, but its result only changes when `stakeholders` changes 

```suggestion
    const hasStakeholders = useMemo(
    () =>
      stakeholders.some(
        stakeholder =>
          stakeholder.accounts &&
          (Array.isArray(stakeholder.accounts)
            ? stakeholder.accounts.length > 0
            : Boolean(stakeholder.accounts)),
      ),
    [stakeholders],
  );
```